src/components/UserInfo.js

```diff
@@ -1,9 +1,10 @@
 class UserInfo {
-  constructor(userNameSelector, userDescriptionSelector) {
+  constructor(userNameSelector, userDescriptionSelector, userAvatarSelector) {
     this._userNameElement = document.querySelector(userNameSelector);
     this._userDescriptionElement = document.querySelector(
       userDescriptionSelector
     );
+    this._userAvatarSelector = document.querySelector(userAvatarSelector);
```

Comment:
`_userAvatarSelector`

Word `selector` means a **string** that is used for searching elements in the DOM